### PR TITLE
[149] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by lorlahdey**
Capture8
Capture9

Kemiojo

### CODE REVIEW

File: submissions/lawalLolade/html-forms/contactFormPage.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                <li><a href="#">Garden</a></li>
+                <li><a href="#">Vases</a></li>
+            </ul>
+        </nav>
+    </div>
+</header>
+    <!-- Header ends -->
+
+
+    <!-- Contact Us Form -->
```
Comment (Ireti203) [inline submissions/lawalLolade/html-forms/contactFormPage.html:27]: Remove whitespaces

File: submissions/lawalLolade/html-forms/contactFormPage.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                    <label for="comment"><strong>Message:</strong></label><br>
+                    <textarea name="comment" id="comments" cols="50" rows="15"></textarea><br><br>
+                </div>
+                <div>
+                    <input type="submit" value="Submit" onclick="alert('Thank you, we will response shortly.')">
+                </div>
+            </fieldset>
+        </form>
+    </div>
+</section>
```
Comment (Ireti203) [inline submissions/lawalLolade/html-forms/contactFormPage.html:64]: Format properly

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/lawalLolade/html-forms/contactFormPage.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+
+    <!-- Footer Begins -->
```
Comment (Ireti203) [inline submissions/lawalLolade/html-forms/contactFormPage.html:67]: Remove whitespaces

File: submissions/lawalLolade/html-forms/contactFormPage.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+<footer>
+    <ul>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+        <li><a href="#">Events</a></li>
+    </ul>
+    <p> &copy; 2020 Flawa. </p>
+</footer>
+    <!-- Footer Ends -->
+
```
Comment (Ireti203) [inline submissions/lawalLolade/html-forms/contactFormPage.html:77]: Remove whitespaces

**Comment by lorlahdey** [comment]
Ireti

File: submissions/lawalLolade/html-forms/contactFormPage.html [hunk 5]
```diff
@@ -5,75 +5,75 @@
...
+                    <div>
+                        <label for="firstname"><strong>First Name:</strong>&nbsp;&nbsp;&nbsp; </label>
+                        <input type="text" name="firstname" id="firstname">
+                    </div><br>
+                    <div>
+                        <label for="lastname"><strong>Last Name:</strong>&nbsp;&nbsp;&nbsp;&nbsp;</label>
+                        <input type="text" name="lastname" id="lastname">
+                    </div><br>
+                    <div>
+                        <label for="email"><strong>Email:</strong>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</label>
```
Comment (Ireti203) [inline submissions/lawalLolade/html-forms/contactFormPage.html:42]: what is the "&nbsp" for? Kindly remove all unnecessary characters.

**Comment by Ireti203** [comment]
There are conflicts on this branch. You should resolve this

**Comment by lorlahdey** [comment]
How can i resolve the conflicts?

File: submissions/lawalLolade/html-forms/contactFormPage.html [hunk 6]
```diff
@@ -0,0 +1,79 @@
+<html lang="en">
+<head>
```
Comment (adaugochi) [inline submissions/lawalLolade/html-forms/contactFormPage.html:2]: Also, I can see that you already have contactFormPage before, Why are you creating a new one?
image

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**